src/dashboards/sm-dns.json

```diff
@@ -919,7 +897,7 @@
       },
       "targets": [
         {
-          "expr": "{probe=~\"$probe\",instance=\"$instance\", job=\"$job\", probe_success=\"0\"}",
+          "expr": "{probe=~\"$probe\", instance=\"$instance\", job=\"$job\", probe_success=\"0\"} | logfmt | line_format \"level={{.level}} target={{.target}} probe={{.probe}} msg={{.msg}}\"",
```

Comment:
[maybe nit]
I can see that this does what the Github issue requested. But I wonder if we lose some value here too. By using `line_format` here you only allow those four items in the log-line to be rendered, so you lose other content such as:
- duration_seconds `… msg="Check failed" duration_seconds=10.000953946`
- err `… msg="Timeout reading from socket" err="read udp 0.0.0.0:46062: raw-read udp 0.0.0.0:46062: i/o timeout"`
- ip `… msg="Resolved target address" ip=68.183.27.198`
- ip_protocol `… msg="Resolving target address" ip_protocol=ip4`
- type `… msg="Beginning check" type=ping timeout_seconds=10`

For me `duration_seconds` and `err` feel like they might be nice ones to try to keep.

There's another visualisation option which we could try as an alternative to using `line_format` here, which is to enable the "Unique labels" option:
![UniqueLabels enabled](https://user-images.githubusercontent.com/2903904/122202299-d3deae80-ce94-11eb-9f85-920ad31962d9.png)

I've created a dashboard and panel with this option enabled, which you can have a play with here:
https://samjewell.grafana.net/d/uQV39cg7k/unique-labels-option-enabled?orgId=1

That way we preserve the extra log content while also showing the unique labels like the Probe location.

Reply:
Aha! This is exactly what I was trying to accomplish. I'll switch to that